test/system/800-config.bats

```diff
@@ -226,8 +226,10 @@ EOF
     FOO="$random_env_var" run_podman --module=$conf_tmp run -d --name=$cname $IMAGE top
     cname="$output"
 
-    # Make sure `env_host` is read
-    run_podman container inspect $cname --format "{{.Config.Env}}"
+    # Make sure `env_host` is read.
+    # Only print the env vars that start with "FOO" to avoid printing output that
+    # may be considered problematic (see run_podman in helpers.bash).
+    run_podman container inspect $cname --format '{{range .Config.Env}} {{if eq "FOO" (slice . 0 3) }} {{.}} {{end}} {{end}}'
```

Comment:
This seems to be part of the wrong commit, it should not be in the codespell commit but rather split it out into a separate one.

Also the slice 0 3 part will fail on a short empty env such as `A=` as it is only two chars long. While one can argue that this is not a valid env the test should not fail because of unrelated host envs because after all that is what you are trying to fix with that.

Reply:
You are right 👍 I will move to a separate commit and make the test more robust before switching the PR to "ready for review".

Reply:
I have added a new commit + changed the template so that it doesn't fail with single letter empty variables.